pyproject.toml

```diff
@@ -23,6 +23,7 @@ dependencies = [
     "s3fs",
     "snowflake-sqlalchemy ==1.2.0",
     "snowflake-connector-python[pandas]",
+    "smart-open[all]>=5.2.1",
```

Comment:
Can you test if we can remove boto3 from these requirements now? smart open might handle that.